scripts/template-version-bump.js

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env node
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Checks git status for template changes and bumps version numbers
+// accordingly. Note that deletions do not cause version numbers to
+// be deleted, to ensure that we have sane metrics in the cases where
+// a deleted template is reinstated by some later commit or only one
+// format of a template is deleted.
+
+'use strict'
+
+const cp = require('child_process')
+const fs = require('fs')
+const path = require('path')
+
+const ROOT_DIR = path.join(__dirname, '..')
+const TEMPLATE_DIR = 'lib/senders/templates'
+const VERSIONS_FILE = '_versions.json'
+const IGNORE = [ VERSIONS_FILE, '_pending.txt', 'index.js', 'README.md' ]
```

Comment:
Run this multiple times and the version file wasn't being ignored on my machine. It added `VersionsEmail` into the file, and incremented each time the script was run.

Are you able to run it multiple times? Possibly be something with my environment?

Reply:
> Are you able to run it multiple times? Possibly be something with my environment?

Oh, whoops, good catch. Moving this back to active while I dig into it.